src/views/admin/KlassenManagementView.vue

```diff
@@ -0,0 +1,415 @@
+    const personenkontexte: Array<UserinfoPersonenkontext> | null = authStore.currentUser?.personenkontexte || [];
+    if (personenkontexte.length > 0) {
+      const matchingOrganisations: UserinfoPersonenkontext[] = personenkontexte.filter(
+        (kontext: UserinfoPersonenkontext) =>
+          organisationStore.allOrganisationen.some((org: Organisation) => org.id === kontext.organisationsId),
+      );
+
+      if (matchingOrganisations.length === 1) {
+        const matchedOrganisation: UserinfoPersonenkontext | undefined = matchingOrganisations[0];
+
+        selectedSchule.value = matchedOrganisation?.organisationsId || null;
+        if (selectedSchule.value) {
+          await organisationStore.getKlassenByOrganisationId(selectedSchule.value);
+          finalKlassen.value = organisationStore.klassen.map((klasse: Organisation) => ({
+            ...klasse,
+            ...getSchuleDetails(klasse),
+          }));
+          hasAutoselectedSchule.value = true;
+        }
+      } else {
+        headers.value.unshift({
+          title: t('admin.schule.dienststellennummer'),
+          key: 'schuleDetails',
+          align: 'start',
+        } as DataTableHeader);
+      }
+    }
+    finalKlassen.value = organisationStore.allKlassen.map((klasse: Organisation) => ({
+      ...klasse,
+      ...getSchuleDetails(klasse),
+    }));
+  });
+</script>
+
+<template>
+  <div class="admin">
+    <h1
+      class="text-center headline"
+      data-testid="admin-headline"
+    >
+      {{ $t('admin.headline') }}
+    </h1>
+    <LayoutCard :header="$t('admin.klasse.management')">
+      <v-row
+        align="center"
+        class="ma-3"
+        justify="end"
+      >
+        <v-col
+          cols="12"
+          md="2"
+          class="py-md-0 text-md-right"
+        >
+          <v-btn
+            class="px-0 reset-filter"
+            data-testid="reset-filter-button"
+            :disabled="!filterActive"
+            @click="resetSearchAndFilter()"
+            size="x-small"
+            variant="text"
+            width="auto"
+          >
+            {{ $t('resetFilter') }}
+          </v-btn>
+        </v-col>
+        <v-col
+          md="3"
+          cols="12"
+          class="py-md-0"
+        >
+          <v-autocomplete
+            autocomplete="off"
+            class="filter-dropdown"
+            :class="{ selected: selectedSchule }"
+            clearable
+            data-testid="schule-select"
+            density="compact"
+            hide-details
+            id="schule-select"
+            ref="schule-select"
+            :items="schulen"
+            item-value="value"
+            item-text="title"
+            :no-data-text="$t('noDataFound')"
+            :placeholder="$t('admin.schule.schule')"
+            required="true"
+            variant="outlined"
+            v-model="selectedSchule"
+            v-model:search="searchInputSchulen"
+            :disabled="hasAutoselectedSchule"
```

Comment:
```suggestion
            autocomplete="off"
            class="filter-dropdown"
            :class="{ selected: selectedSchule }"
            clearable
            data-testid="schule-select"
            density="compact"
            :disabled="hasAutoselectedSchule"
            hide-details
            id="schule-select"
            :items="schulen"
            item-value="value"
            item-text="title"
            :no-data-text="$t('noDataFound')"
            :placeholder="$t('admin.schule.schule')"
            ref="schule-select"
            required="true"
            variant="outlined"
            v-model="selectedSchule"
            v-model:search="searchInputSchulen"
```
Alphabetisch übersichtlicher bei so vielen props